cpp/include/cudf/io/parquet.hpp

```diff
@@ -990,6 +1006,8 @@ class chunked_parquet_writer_options {
   /**
    * @brief Returns maximum page size, in rows. If set larger than the row group size, then this
    * will return the row group size.
```

Comment:
```suggestion
   * @brief Returns maximum page size, in rows. 
   *
   * If set larger than the row group size, then this will return the row group size.
```